cms/templates/container_chromeless.html

```diff
@@ -0,0 +1,279 @@
+## coding=utf-8
```

Comment:
[request] Please add some sort of comment to this file to explain what it is, why it exists, and how it's intended to be different from the template file it's based off of.

Reply:
Added comment with description